src/cirrus/delegate.py

```diff
@@ -83,7 +84,13 @@ def main():
             exit_code = 0
         else:
             command_path = "{0}/bin/{1}".format(home, args[0])
-            exit_code = run_command([command_path, ] + args[1:])
+            if not os.path.exists(command_path):
+                msg = "Unknown command: {}".format(args[0])
+                print msg
+                print format_help(commands)
+                exit_code = 1
```

Comment:
May want to exit with 127 for command not found? Nothing wrong with 1 on the other hand.